tests/test_openjdk.py

```diff
@@ -174,7 +174,7 @@ class TestExtendedParams:
                 expected_strings=["Iteration: (2)"],
                 expected_err_strings=["OutOfMemoryError"],
                 expected_exit_status=[1],
-                java_params="-Xmx10M",
+                java_params="-Xmx10M -Xlint:unchecked",
```

Comment:
Could you please explain in the commit message what & why you have done this change?